astropy/tests/disable_internet.py

```diff
@@ -1,10 +1,7 @@
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
 
 import contextlib
 import socket
-
-from ..extern.six.moves import urllib
+import urllib.request
```

Comment:
How come a `urrlib.request` can replace `urllib`?

Reply:
What do you mean? `urllib` in Python 2 was split up. Can you be more specific?

Reply:
technically this wasn't needed, but @saimn instinct was right that we should only import this modul from urllib as we don't use any of the others...

Reply:
Thanks for the explanation. I was actually just scanning for obvious inconsistencies, I haven't had the motivation to investigate them in details.